backend/devices/bitbox02/keystore_simulator_test.go

```diff
@@ -412,12 +419,48 @@ func makeTx(t *testing.T, device *Device, recipient *maketx.OutputInfo) *btc.Pro
 		GetPrevTx: func(chainhash.Hash) (*wire.MsgTx, error) {
 			return prevTx, nil
 		},
+		GetAddress: func(scriptHashHex blockchain.ScriptHashHex) *addresses.AccountAddress {
+			for _, address := range addrsInDifferentAccount {
+				if address.PubkeyScriptHashHex() == scriptHashHex {
+					return address
+				}
+			}
+			return nil
+		},
 		Signatures: make([]*types.Signature, len(txProposal.Transaction.TxIn)),
 		FormatUnit: coinpkg.BtcUnitDefault,
 	}
 
 }
 
+func TestSimulatorSignBTCTransactionSendSelfDifferentAccount(t *testing.T) {
+	testInitializedSimulators(t, func(t *testing.T, device *Device, stdOut *bytes.Buffer) {
+		t.Helper()
+
+		cfg := makeConfig(t, device, signing.ScriptTypeP2WPKH, mustKeypath("m/84'/0'/1'"))
+		selfAddress := addresses.NewAccountAddress(
+			cfg,
+			signing.NewEmptyRelativeKeypath().Child(0, false).Child(0, false),
+			network,
+			log)
+		proposedTransaction := makeTx(t, device, maketx.NewOutputInfo(selfAddress.PubkeyScript()))
+
+		require.NoError(t, device.Keystore().SignTransaction(proposedTransaction))
+		require.NoError(t, proposedTransaction.Finalize())
+		require.NoError(
+			t,
+			btc.TxValidityCheck(
+				proposedTransaction.TXProposal.Transaction,
+				proposedTransaction.TXProposal.PreviousOutputs,
+				proposedTransaction.TXProposal.SigHashes()))
+
+		// Before simulator v9.20, address confirmation data was not written to stdout.
```

Comment:
v9.20 in comment, but v9.22 in the if check below. What about v9.20 and v9.21?

Reply:
This is actually a leftover comment from a copy-paste :)